account/repository/identity.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 const (
-	// KeycloakIDP is the name of the main Keycloak Identity Provider
-	KeycloakIDP string = "kc"
+	// OAuthServiceIDP is the name of the main OAuthService Identity Provider
+	OAuthServiceIDP string = "kc"
```

Comment:
I find the name `OAuthServiceIDP` a bit confusing as it mixes a protocol with a service ("identity provider"). May I suggest that we name it to something like `DefaultIDP` instead ? or maybe `OSIOIdentityProvider`, as a reference to the main/internal IDP compared to external IDPs such as GitHub, etc. ? 
(this also joins @alexeykazakov's comment in https://github.com/fabric8-services/fabric8-auth/pull/594/files#r209036273)